packages/expression-parser/src/expression-parser/ExpressionParser.js

```diff
@@ -15,11 +15,18 @@ import { create, all } from 'mathjs';
  * //Evaluate an expression that has a as a variable
  * expressionParser.evaluate('a + 2'); //returns 3
  */
+
+const average = (...argumentList) => {
+  const existingValues = argumentList.filter(a => a !== 'undefined');
+  const sum = existingValues.reduce((a, b) => a + b, 0);
+  return sum / existingValues.length;
+};
 export class ExpressionParser {
   constructor() {
     this.math = create(all, {});
     this.parser = this.math.parser();
-    this.importFunctions();
+    this.customFunctions = this.getCustomFunctions();
+    this.math.import(this.customFunctions);
```

Comment:
Hopefully this doesn't add too much overhead, but because of past [performance concerns](https://github.com/beyondessential/tupaia-backlog/issues/1763), could we take 2-3 measurements to verify it?

Just measuring how long it takes it create a single ExpressionParser instance should be enough:
```js
const a = Date.now();
new ExpressionParser();
console.log(`${Date.now() - a{)ms`);
```

Reply:
Just had a try, it varies around 19ms to 40ms

Reply:
This is acceptable, thanks!